<launch>

    <arg name="device" default="can0" doc="Communication port of charger for canbus interface use can0" />
    <arg name="node_id" default="10" doc="CANOpen node ID, default 10" />
    <arg name="comm_interface" default="canbus" doc="Communication interface of charger, use modbus/canbus/gpio " />

    <!-- Xnergy Charger Node -->
    <node pkg="xnergy_charger_rcu" type="xnergy_charger_node.py" name="xnergy_charger_rcu" output="screen">
        <param name="communication_interface" value="$(arg comm_interface)" />
        <param name="device" value="$(arg device)" />
        <param name="node_id" value="$(arg node_id)" />
    </node>

</launch>